### PR TITLE
File for Yuge Shen and Shengqing Xia

### PR DESCRIPTION
**Pull request by yuge97**

### CODE REVIEW

File: Multivariable_Regression_Tutorial.Rmd [hunk 1]
```diff
@@ -0,0 +1,183 @@
+# Tutorial for Multivariable Linear Regression
+
+Yuge Shen, Shengqing Xia
+
+```{r setup, include=FALSE}
```
Comment (bl2788) [inline Multivariable_Regression_Tutorial.Rmd]: because you have a setup trunk, please remove the setup label.

**Review by bl2788 — COMMENTED** [review]
1.please remove the setup label in your setup trunk
2.be aware that your png file's path may change when resources/<project_name>/ is created.

**Review by bl2788 — CHANGES_REQUESTED** [review]
1.remove the setup label in your setup trunk
2.be aware that your png file path may change when resources/<project_name>/ is created.

**Review by bl2788 — CHANGES_REQUESTED** [review]
You don't need to remove all the trunk, just remove the setup label itself.

**Review by bl2788 — CHANGES_REQUESTED** [review]
Sorry for the misunderstanding, it should be like {r , include=FALSE}

**Review by bojum — CHANGES_REQUESTED** [review]
@yuge97 Your content will look nicer if you remove one # from each of the # headers except for the first line. So that the rendered page wouldn't skip ##.

**Comment by bojum** [comment]
🎉 Merged. Thanks for your contribution!

**Pull request merged**